e2e/autoscript-apps/src/authn-webauthn/autoscript.ts

```diff
@@ -61,8 +61,8 @@ function autoscript() {
       rxDelay(delay),
       mergeMap((step) => {
         console.log('Choose Passwordless login');
-        const cb = step.getCallbackOfType('ChoiceCallback');
-        cb.setChoiceIndex(0);
+        const cb = step.getCallbackOfType('ConfirmationCallback');
```

Comment:
has the test journey change and send a confirmation callback?

Reply:
The test journey was removed, so I duplicated a journey that Stoyan uses, and it had a Confirmation Callback instead. This is a manual test, and not automation, so changes here don't affect CI.